app/webpack.config.js

```diff
@@ -295,6 +295,22 @@ module.exports = [
           test: /\.(jpe?g|png|gif|ico|eot|ttf|map|woff2?)(\?v=\d+\.\d+\.\d+)?$/i,
           type: 'asset/resource',
         },
+        // Fix WebRTC buffered transmission: https://github.com/yjs/y-webrtc/pull/25
+        {
+          test: /y-webrtc\.js$/,
+          loader: 'string-replace-loader',
+          options: {
+            search: 'simple-peer/simplepeer.min.js',
+            replace: path.join(
+              '..',
+              '..',
+              '..',
+              'app',
+              'vendor',
+              'SimplePeerExtended.js'
+            ),
+          },
+        },
```

Comment:
Initially tried absolute path to `SimplePeerExtended.js` here, but could not make it work on Windows. This ugly relative path (assuming node_modules location) might just work.

Reply:
Still broken. Is it really that `path.join()` does not join parts with `path.sep` on Windows :flushed:

Reply:
Or maybe `string-replace-loader` breaks Windows `path.sep` on replace.